R/start_transcription.R

```diff
@@ -27,14 +50,38 @@ function(
     format = tools::file_ext(url),
     language = "en-US",
     hertz = NULL,
+    output_bucket = NULL,
+    channel_identification = NULL,
+    show_speaker_labels = NULL,
+    max_speaker_labels = NULL,
+    vocabulary_name = NULL,
     ...
 ) {
     bod <- list(Media = list(MediaFileUri = url))
     bod$MediaFormat <- format
     bod$LanguageCode <- language
+    bod$TranscriptionJobName <- name
     if (!is.null(hertz)) {
         bod$MediaSampleRateHertz <- hertz
     }
-    bod$TranscriptionJobName <- name
+    if (!is.null(output_bucket)) {
```

Comment:
You don't need to check whether it's NULL. Nothing happens when you had `NULL` to a list using `<-`. You can check: `li <- list() ; li$a <- NULL ; li` will show an empty list.